OpenDreamClient/Rendering/DreamViewOverlay.cs

```diff
@@ -45,6 +47,7 @@ sealed class DreamViewOverlay : Overlay {
     private Stack<RendererMetaData> _rendererMetaDataRental = new();
     private Stack<RendererMetaData> _rendererMetaDataToReturn = new();
     private Matrix3 _flipMatrix;
+    private const LookupFlags Flags = LookupFlags.Approximate | LookupFlags.Dynamic | LookupFlags.Static | LookupFlags.Sundries | LookupFlags.Contained;
```

Comment:
I don't know if OD uses the engine's containers. If it doesn't (or only uses it for non-visible entities), then the `LookupFlags.Contained` should be removed, which should speed things up a bit.

Reply:
We don't. Would be a good idea to use them, though.

Reply:
I'll just remove the flag then.